testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7584,6 +7584,20 @@ private static void checkArrayConcatAggFuncFails(SqlOperatorFixture t) {
     f.checkFails("^Array[]^", "Require at least 1 argument", false);
   }
 
+  @Test void testArrayFunctionConstructor() {
```

Comment:
rename test to `testArrayFunction`. it's not a constructor. 

add javadoc making clear that it's for the function in the Spark library.

test what happens when the Spark library is not enabled. (as I said elsewhere, I think you probably don't need the conformance setting.)